content/concepts/http2.textile

```diff
@@ -32,40 +30,32 @@ jump_to:
     - Conclusion#conclusion
 ---
 
-h1(#http-2). HTTP/2: A Conceptual Deep Dive
+The Internet as we know it would not have been possible without HTTP (Hypertext Transfer Protocol). HTTP has come a long way since its inception when it started as a simple one-line ASCII string protocol. HTTP/2 is a significant version upgrade over it predecessor HTTP/1.1
 
-The Internet, as we know it, would not have been possible without Hypertext Transfer Protocol or HTTP. HTTP has come a long way since its inception. HTTP started as a simple one-line ASCII string protocol. HTTP/2 is a significant version upgrade over it predecessor HTTP/1.1
+This article covers the limitations of HTTP/1.1 and how those limitations led to the protocol’s evolution to HTTP/2. The article delves into the technical concepts behind HTTP/2 and touches upon some of its own limitations.
 
-This article covers the limitations of HTTP/1.1 and how those limitations led to the protocol’s evolution to HTTP/2. The article delves into the technical concepts behind HTTP/2 and touches upon some of its limitations.
+h2(#history). Background: A Brief History of HTTP
 
-h2(#history). Background - A Brief History of HTTP
-
-h2(#origin). A Humble Origin
+h2. Humble origins
```

Comment:
This double `h2` bothers me, and when checking out the docs locally it looks terrible and on the preview app it doesn't look good either.

How about a single header, "Humble Origins - A Brief History of HTTP"?

Reply:
done